ui.frontend/src/main/components/App/App.js

```diff
@@ -35,12 +35,13 @@ import partialConfig from './config';
 import '../../site/main.scss';
 
 const App = () => {
-    const { storeView, graphqlEndpoint } = document.querySelector('body').dataset;
+    const { storeView, graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
```

Comment:
I'm wondering whether we should make it more explicit that this setting affects only the queries and not the mutations...

Reply:
@dplaton good point. Where do you think would be the best place to document this?

Reply:
For the moment we can only leave a comment in the code. Sadly, we don't have any clear documentation now.